task-2/src/store/allReducers/rootReducer.js

```diff
@@ -0,0 +1,33 @@
+import {
```

Comment:
Consider renaming the containing folder to "reducers".

Reply:
File name changed!